resources/api-docs/1.0/entities.md

```diff
@@ -192,3 +195,86 @@ With each request to an object (ie. `character`, `location`, etc), you can inclu
 ```
 
 Notice the new array objects `attributes`, `entity_files`, `entity_events`, `posts`, `entity_abilities` and `relations`.
+
+<a name="deleted-entities"></a>
+## Deleted Entities
+
+You can view the recoverable deleted entities on the `/recovery` endpoint
+
+### Result
+
+```json
+{
+ "data": [
+        {
+            "id": 2,
+            "name": "Thaelia",
+            "type": "location",
+            "type_id": 3,
+            "child_id": 2,
+            "tags": [],
+            "is_private": false,
+            "is_template": false,
+            "campaign_id": 1,
+            "is_attributes_private": false,
+            "tooltip": null,
+            "header_image": null,
+            "image_uuid": null,
+            "created_at": "2023-08-22T20:01:48.000000Z",
+            "created_by": null,
+            "updated_at": "2023-08-22T23:19:07.000000Z",
+            "updated_by": 1,
+            "urls": {
+                "view": "http://app.kanka.test:8081/w/1/entities/2",
+                "api": "http://api.kanka.test:8081/1.0/campaigns/1/locations/2"
+            }
+        },
+        {
+            "id": 23,
+            "name": "Middle Earth",
+            "type": "location",
+            "type_id": 3,
+            "child_id": 16,
+            "tags": [],
+            "is_private": false,
+            "is_template": false,
+            "campaign_id": 1,
+            "is_attributes_private": false,
+            "tooltip": null,
+            "header_image": null,
+            "image_uuid": null,
+            "created_at": "2023-08-22T20:22:21.000000Z",
+            "created_by": null,
+            "updated_at": "2023-08-22T23:19:07.000000Z",
+            "updated_by": null,
+            "urls": {
+                "view": "http://app.kanka.test:8081/w/1/entities/23",
+                "api": "http://api.kanka.test:8081/1.0/campaigns/1/locations/16"
+            }
+        }
+    ],
+}
+```
+
+<a name="recover-entities"></a>
+## Recover Deleted Entities
+
+You can post an array with the ids of the entities you want to recover to the `/recover` endpoint to undo the deletion (this is a boosted/premium only feature).
+
+| Parameter | Type | Description
+| :- | :- | :- |
+| `entities` | `array` | The ids of the entities to recover. |
+
+### Examples
+
+| Method | Body | Headers |
+| :- |   :-   |  :-  |
+| POST | `"entities": [2,132,10]` | Default |
```

Comment:
We don't usually provide examples since we expect people to already know how to interact with an API. However the `Method|Uri|Headers` table is missing before the body. Please reproduce the doc layout from other files to keep it consistent.